tests/acceptance/features/useridentity-POST-UnhappyPath.feature

```diff
@@ -55,3 +55,36 @@ Feature: UserIdentity Post - Happy Path
     Then the status code should be 200
     And the stored identity should be returned
     And the stored identity isValid field is false
+
+
+  Scenario: A stored identity is returned for the user, and kidValid is false
+    Given I have a user with a Stored Identity, with "invalid" kid
+    When I make a request for the users identity with a VTR "P2"
+    Then the status code should be 200
+    And the untrusted stored identity should be returned
+    And the stored identity kidValid field is false
+    And the stored identity signatureValid field is false
+
+  Scenario: A stored identity is returned for the user, and kidValid is false
```

Comment:
These scenario names are all the same, could you make them reflect the different scenarios. E.g. "no kid" vs "invalid kid"